test/source/tests/compose.ts

```diff
@@ -2661,8 +2661,10 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
         const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, acct);
         await SetupPageRecipe.autoSetupWithEKM(settingsPage);
         const composePage = await ComposePageRecipe.openStandalone(t, browser, acct);
-        await ComposePageRecipe.fillMsg(composePage, { to: 'mock.only.pubkey@flowcrypt.com' }, 'no valid key');
-        await ComposePageRecipe.waitForToastToAppearAndDisappear(composePage, 'Draft not saved: Error: Your account keys are revoked');
+        await Promise.all([
+          ComposePageRecipe.fillMsg(composePage, { to: 'mock.only.pubkey@flowcrypt.com' }, 'no valid key'),
+          ComposePageRecipe.waitForToastToAppearAndDisappear(composePage, 'Draft not saved: Error: Your account keys are revoked'),
```

Comment:
This toast appears only once, doesn't reappear on further edits, is this behavior expected?

Reply:
Yes, it's expected behavior - we set `disableSendingDrafts` to `true` in case of such error (as there is no reason to retry draft saving)

https://github.com/FlowCrypt/flowcrypt-browser/blob/master/extension/chrome/elements/compose-modules/compose-draft-module.ts#L183